kong/api/api_helpers.lua

```diff
@@ -478,7 +478,9 @@ function _M.attach_new_db_routes(app, routes)
         return method_handler(self, kong.db, handler_helpers)
       end
 
-      methods[method_name] = parse_params(wrapped_handler)
+      if route_path ~= "/workspaces" or method_name ~= "POST" then
```

Comment:
what is the purpose of this judgment here?

Reply:
We do not want the lapis to listen for "POST /workspaces" operation.